brainstorm/src/components/items.tsx

```diff
@@ -78,15 +98,14 @@ export function ItemsView(props: {
 	}
 }
 
-/**
- * Removes a node from its parent {@link Items}.
- * If the note is not in an {@link Items}, it is left unchanged.
- */
-export function deleteItem(item: Item): void {
+export interface ItemParent {
+	deleteItem(item: Item): void;
+}
+
+export function deleteItemFromParent(item: Item & TreeNode): void {
```

Comment:
Might make more sense to expose a `tryAsItemParent(node: TreeNode)` which just reads the symbol off a node, returning ItemParent | undefined. This will require less boilerplate as we ad more methods to ItemParent